x-pack/plugins/security_solution/public/resolver/view/resolver_no_process_events.tsx

```diff
@@ -49,7 +49,7 @@ export const ResolverNoProcessEvents = () => (
     </EuiText>
     <EuiSpacer size="m" />
     <StyledEuiCodeBlock language="html" paddingSize="s" isCopyable>
-      {"event.category: 'process'"}
+      {'event.category: "process"'}
```

Comment:
There wasn't a bug filed for this yet, but when copying the text into the search bar it doesn't work unless you use double quotes.